josm-presets/de-signals-eso.xml

```diff
@@ -38,6 +37,26 @@ See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 			de.display_values="In Wegrichtung,Entgegen Wegrichtung,Beide Richtungen"
 			/>
 	</chunk>
+	<chunk id="traversable">
+		<multiselect key="railway:signal:traversable:type"
+			text="Traversable: action at stop"
+			de.text="Mastschild: Verhalten bei Halt"
+			delimiter="|"
```

Comment:
I tested your patch and found out that JOSM XXXXX is unable to handle the `|` delimiter properly.

![error](https://user-images.githubusercontent.com/3611273/150186604-0a058321-3ec2-4c5a-a3e6-008d9f1409c3.png)


However, a `#` works:

```diff
diff --git a/josm-presets/de-signals-eso.xml b/josm-presets/de-signals-eso.xml
index 93d69e0..42c4e46 100644
--- a/josm-presets/de-signals-eso.xml
+++ b/josm-presets/de-signals-eso.xml
@@ -41,10 +41,10 @@ See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
                <multiselect key="railway:signal:traversable:type"
                        text="Traversable: action at stop"
                        de.text="Mastschild: Verhalten bei Halt"
-                       delimiter="|"
-                       values="written_default_speed;dictated_default_speed|on_no_contact_on_sight;written_default_speed;dictated_default_speed|always_on_sight"
-                       display_values="white-red-white, red-triangle on white, or pure red|white-yellow-white-yellow-white|white-black-white-black-white"
-                       de.display_values="weiß-rot-weiß, rotes Dreieck auf weiß oder nur rot|weiß-gelb-weiß-gelb-weiß|weiß-schwarz-weiß-schwarz-weiß"
+                       delimiter="#"
+                       values="written_default_speed;dictated_default_speed#on_no_contact_on_sight;written_default_speed;dictated_default_speed#always_on_sight"
+                       display_values="white-red-white, red-triangle on white, or pure red#white-yellow-white-yellow-white#white-black-white-black-white"
+                       de.display_values="weiß-rot-weiß, rotes Dreieck auf weiß oder nur rot#weiß-gelb-weiß-gelb-weiß#weiß-schwarz-weiß-schwarz-weiß"
                        default="written_default_speed;dictated_default_speed"
                        />
                <combo key="railway:signal:traversable"
```

@DerDakon Should we work around this JOSM bug and merge the pull request? I can apply my bug fix patch during merge (separate commit).

Reply:
You can't without tricks, you need to have in an approved branch first…